python/cuml/cuml_accel_tests/upstream/umap/xfail-list.yaml

```diff
@@ -0,0 +1,43 @@
+- reason: Test fails with cuml.accel
```

Comment:
I haven't done much test diagnosing or grouping _yet_. Just getting the infra setup in this first PR.